### ISSUE TITLE
subscribe to all replies to a note

### ISSUE THREAD

**Issue by adrienlacombe** [PrimalHQ/primal-android-app#252]
## Summary
I would like to be notified for all replies to a note

### Basic example
Someone ask an interesting question and I would like to quickly see the answers

### Motivation
I haven't found a nice workaround for this unfortunately, except maybe copy pasting the link to then note and open it frequently, this is painful

Thank you

**Comment by bitcoinuser** [comment]
Agree. Duplicate of: https://github.com/PrimalHQ/primal-android-app/issues/60

**Comment by adrienlacombe** [comment]
> Agree. Duplicate of: #60

yes sorry
